starknetup.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/sh
```

Comment:
nit: please use spaces instead of tabs in bash scripts. By default, two spaces should be used for an indent (but if you prefer 4 spaces like in python, this is also acceptable imo).